### PR TITLE
[script.module.vici]5.5.3

### PR DESCRIPTION
**Pull request by venus36**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by venus36** [comment]
@Rechi 
this module is for the strongswan
I am going to create addon for IKEv2 VPN, this addon will use the strongswan and this module
so we need to install this module automatically from our addon dependency

**Comment by Rechi** [comment]
> why do you add `No addon is using it`?

Because currently no addon is using this module.

> can you please merge it?

Not yet, first you have to submit the addon which uses this module. We do not want unsed modules in our repo.

**Comment by learningit** [comment]
We do not allow VPN add-ons in the kodi.org repo.

File: script.module.vici/addon.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.vici" name="vici" version="5.5.3" provider-name="venus36">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0"/>
+  </requires>
+  <extension point="xbmc.python.module" library="lib" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en">Python StrongSwan Library</summary>
```
Comment (enen92) [inline script.module.vici/addon.xml]: en_GB please

**Comment by enen92** [comment]
Closed as the PR is stalled

**Pull request closed without merging**